content/00-first-steps/02-wallet-quickstart.mdx

```diff
@@ -50,14 +43,14 @@ Fedora 28
 64-bit dual core processors  
 4 GB of RAM  
 at least 8GB disk space for IOHK and Mordor  
-At least 50gb for FastSync  
+At least GB for FastSync  
```

Comment:
Is missing a number :)

Reply:
Thanks Leonor. Everyone else missed this.